variables.tf

```diff
@@ -79,7 +79,7 @@ variable "env" {
 variable "ingress_settings" {
   description = "Ingress settings can be one of 'all', 'internal', 'internal-and-cloud-load-balancing'(default)."
   type        = string
-  default     = "internal-and-cloud-load-balancing"
+  default     = "INGRESS_TRAFFIC_INTERNAL_LOAD_BALANCER"
```

Comment:
Ils ont l'air d'avoir changé les noms :
Control traffic that reaches the cloud function. Allowed values are ALLOW_ALL, ALLOW_INTERNAL_AND_GCLB and ALLOW_INTERNAL_ONLY .